server/services/reportService.ts

```diff
@@ -0,0 +1,46 @@
+import { Connection, getConnection, InsertResult } from 'typeorm'
+
+import Report from '../repositories/entities/report'
+import ReportDefinition from '../repositories/entities/reportDefinition'
+
+export interface IReport {
+  id?: string
+  reportDefinitionId: number
+}
+
+export default class ReportService {
```

Comment:
In all the tests I've seen this service is mocked out, are there any tests that test that data is persisted?

Reply:
Ack, had the initial UI screen creating reports but decided that'd be bad right now so removed it. Forgot to include the test elsewhere. I'll get on it.

Reply:
Added a few integration tests to run through the full stack